tests/test_collection_api.py

```
@@ -0,0 +1,113 @@
+        user_id=user.id,
+        created_at=datetime.now(UTC),
+    )
+    async_db.add(thread)
+    await async_db.commit()
+    await async_db.refresh(thread)
+
+    return thread
+
+
+@pytest.mark.asyncio
+async def test_create_collection(auth_client: AsyncClient, async_db: AsyncSession) -> None:
```

Ruff (0.15.1)

[warning] Unused function argument: `async_db`

(ARG001)